<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowseComment</class>
 <widget class="QDialog" name="BrowseComment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查看评论</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>185</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>back.png</normaloff>back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>185</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>next.png</normaloff>next.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>275</y>
     <width>400</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>评价星级</string>
   </property>
   <widget class="QLabel" name="lbls3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbls5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbls4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbls2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbls1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbContent">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>375</y>
     <width>400</width>
     <height>350</height>
    </rect>
   </property>
   <property name="title">
    <string>评价内容</string>
   </property>
   <widget class="QTextEdit" name="teContent">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>350</width>
      <height>300</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblDate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>225</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>175</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblbrcommBGN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>750</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lblbrcommBGN</zorder>
  <zorder>gbContent</zorder>
  <zorder>btnBack</zorder>
  <zorder>btnNext</zorder>
  <zorder>groupBox</zorder>
  <zorder>lblDate</zorder>
  <zorder>lblTitle</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnBack</sender>
   <signal>clicked()</signal>
   <receiver>BrowseComment</receiver>
   <slot>BackOne()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNext</sender>
   <signal>clicked()</signal>
   <receiver>BrowseComment</receiver>
   <slot>NextOne()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BackOne()</slot>
  <slot>NextOne()</slot>
 </slots>
</ui>
